### ISSUE TITLE
Letsencrypt Root Certificate missing

### ISSUE THREAD

**Issue by Messinger** [nextcloud/server#17427]
I tried to integrate a remote nextcloud storage into my own instance. But always get an error message that is not possible (both version 16.0.5). After a while I saw, that I have a ssl problem. Both sides has a Lets Encrypt Certificate. 

So I downloaded the current X3 root certificate from letsencrypt (https://letsencrypt.org/certificates/) and did on console a 

```bash
./occ security:certificates:import `pwd`/lets-encrypt-x3-cross-signed.pem.txt
```

Voila. I could insert the remote storage.

Removing the cert it wasn't possible again. 

So - why are the Letsencrypt Root Certicates not trusted out of the box. May from system around. IMHO letsencrypt is meanwhile the most important certificate issuer for private users - and Nextcloud IS for private users.

IMHO this is a bug. Every browser, every system meanwhile accepts Letsencrypt out of the box. We don't talk about an old version of nextcloud but about the latest version. 

Please enroll letsencrypt root certificates with Nextcloud so interconnect between LE secured instances will work in future out of the box.

**Comment by johkoenig** [comment]
The "ISRG Root X1" certificate - whch is the LE root cert - is part of nextclouds certificate storage, for quite some time now (I checked 15,16, 17 and master). 
So, this bug seems to be related to a chain of trust problem than a missing certificate.

**Comment by Messinger** [comment]
Ok. I researched a while, every browser, curl etc. pp. were fine (and showed me the complete trustchain) of this certificate. 

After a while I detected that openssl had problem - and yes. The nextcloud instance serving the share had a broken cert, which were accepted by browsers and so but not by nextcloud (and openssl). 

So, you're right. The ISRG is fairly enough. Seems it was a very very strange hidden error. 

Sorry and thanks for your patience.
